lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -507,9 +507,33 @@ def add_managed_billing(xml, options)
       end
 
       def add_stored_credentials(xml, parameters)
-        xml.tag! :MITMsgType, parameters[:mit_msg_type] if parameters[:mit_msg_type]
-        xml.tag! :MITStoredCredentialInd, parameters[:mit_stored_credential_ind] if parameters[:mit_stored_credential_ind]
-        xml.tag! :MITSubmittedTransactionID, parameters[:mit_submitted_transaction_id] if parameters[:mit_submitted_transaction_id]
+        return unless parameters[:mit_stored_credential_ind] && parameters[:mit_stored_credential_ind] == 'Y' || parameters[:stored_credential] && !parameters[:stored_credential].values.all?(&:nil?)
```

Comment:
Can we simplify 
`parameters[:mit_stored_credential_ind] && parameters[:mit_stored_credential_ind] == 'Y'` 
to
`parameters[:mit_stored_credential_ind] == 'Y'`?

Reply:
I'm less sure about this, I'll test it out with `.dig`.

Reply:
I don't think we need `.dig` because we aren't dealing with a nested hash. The result of `parameters[:mit_stored_credential_ind]` if the key `mit_stored_credential_ind` is missing is `nil`.

Reply:
You're right; I had tried it that way previously and it failed for some reason, no forgotten, but this is good.